src/cli/pm_trusted_command.zig

```diff
@@ -263,14 +268,19 @@ pub const TrustCommand = struct {
                     const resolution = &resolutions[package_id];
                     var package_scripts = scripts[package_id];
 
-                    if (try package_scripts.getList(
+                    const maybe_scripts_list = package_scripts.getList(
                         pm.log,
                         pm.lockfile,
                         node_modules_dir,
                         abs_node_modules_path.items,
                         alias,
                         resolution,
-                    )) |scripts_list| {
+                    ) catch |err| {
+                        if (err == error.ENOENT) continue;
+                        return err;
+                    };
```

Comment:
Is the granularity of the error too large. What I mean is that if one of the scripts fails to have a file does getting the list `getList` fail for all? Should the `error.ENOENT` be done at the [`getList`](https://github.com/oven-sh/bun/blob/e2567512188b3d7e9926b8716767bebac9488619/src/install/lockfile.zig#L2620) level or maybe a new method `.getListOnDisk`